spec/cypress/app_commands/activerecord_fixtures.rb

```diff
@@ -0,0 +1,22 @@
+# you can delete this file if you don't use Rails Test Fixtures
+
+fixtures_dir = command_options.try(:[], 'fixtures_dir')
+fixture_files = command_options.try(:[], 'fixtures')
+
+if defined?(ActiveRecord)
+  require "active_record/fixtures"
+
+  fixtures_dir ||= ActiveRecord::Tasks::DatabaseTasks.fixtures_path
+  fixture_files ||= Dir["#{fixtures_dir}/**/*.yml"].map { |f| f[(fixtures_dir.size + 1)..-5] }
```

Comment:
Uhh, what does this do?

In particular, 
```
.map { |f| f[(fixtures_dir.size + 1)..-5] }
```
looks... not very readable.

Reply:
No idea - this was added by cypress-on-rails